platform/dist/axios.js

```diff
@@ -112,12 +112,11 @@ function stepExport(step, message, key) {
     if (step) {
         if (step.export) {
             step.export(key, message);
+            return;
```

Comment:
Using this `return`, the `console.log` will not be called in this case, just confirming: this is the idea?